modules/xquare-user-interfaces/src/components/notice/noticeitem/noticeitem.tsx

```
@@ -0,0 +1,49 @@
+  width: 90%;
+
+  font-size: 15px;
+  font-weight: 500;
+  color: ${Xquare_colors.purple[500]};
+
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const Date = styled.div`
```

Biome (2.1.2)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)